scilpy/io/streamlines.py

```diff
@@ -144,3 +144,33 @@ def load_tractogram_with_reference(parser, args, filepath,
         parser.error('{} is an unsupported file format'.format(filepath))
 
     return sft
+
+
+def filter_tractogram_data(tractogram, streamline_ids):
+    """ Filter tractogram data (data_per_streamline, data_per_point)
+    according to streamline ids
+
+    Parameters:
+    -----------
+    tractogram: Tractogram or StatefulTractogram
+        Tractogram containing the data to be filtered
+    streamline_ids: array_like
```

Comment:
Array like is fine. But make sure in the code to convert it to numpy, cast it as int, make sure all indices exists (or else logging something)